### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by liunuaa** [tensorflow/nmt#238]
I have updated the version of tensorflow to nightly 1.6 and python to 3.5. However, when I run the scripts,  a problem occurred. Anyone has the similar problem?

![image](https://user-images.githubusercontent.com/22492462/34863984-12a50686-f7ad-11e7-8a04-722a1b6b3c41.png)

**Comment by nave01314** [comment]
What parameters are you running with?

**Comment by liunuaa** [comment]
@nave01314   I have just test the code, so I run the parameter supplied.
python -m nmt.nmt \
    --attention=scaled_luong \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_attention_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu

**Comment by oahziur** [comment]
@liunuaa Can you try clean up `/tmp/nmt_attention_model ` and run the command again? Maybe there is some stale hparams file in your `out_dir`.

**Comment by liunuaa** [comment]
@oahziur Ok, thanks, I will try.

**Comment by lmthang** [comment]
Have you resolved the issue? Close for now.
